src/components/ListView.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import ListCard from '../components/ListCard';
+import { withStyles } from '@material-ui/core/styles';
+import axios from 'axios';
+import CircularProgress from '@material-ui/core/CircularProgress';
+
+const styles = theme => ({
+  container: {
+    backgroundColor: 'grey',
+    display: 'flex',
+    alignItems: 'center',
+    flexDirection: 'column',
+    height: '100%',
+    justifyContent: 'center'
+  }
+});
+
+class ListView extends React.Component {
+
+  state = {reports: null};
+
+  componentDidMount() {
+    axios.get(process.env.REACT_APP_GET_REPORTS)
+      .then(reports => {
+        this.setState({...this.state, reports: reports.data});
+      })
+      .catch(error => error);
+  }
+
+  render() {
+    const {reports} = this.state;
+    const {classes} = this.props;
+    return (
+    <div className={classes.container}>
+    {
```

Comment:
I'd recommend just pulling this out to the top above the return statement as :

` if (!reports) {
return <CircularProgress/>;
}` 

then you can kill the ternary operator.

I think it's fine that the spinner is at the top.